mappings/net/minecraft/world/BlockView.mapping

```diff
@@ -10,6 +10,7 @@ CLASS net/minecraft/class_1158 net/minecraft/world/BlockView
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD method_3774 getBlockAt (III)Lnet/minecraft/class_197;
```

Comment:
shouldn't this be `getBlockState`? unless it returns a Block in which case it should be `getBlock`

Reply:
Pretty sure that method is for versions before BlockState existed